.dockerignore

```diff
@@ -5,6 +5,7 @@
 **/CMakeLists.txt
 *.iml
 bin
+!bin/dep
```

Comment:
i think you need to add `.dep` to this file, otherwise `bin/dep` will try to run the `.dep` executable built on the host system.

when i build go-deps, i get this error:
```bash
$ docker build -t gcr.io/runconduit/go-deps:sigtest -f Dockerfile-go-deps .
Sending build context to Docker daemon  91.55MB
...
Step 4/9 : RUN bin/dep ensure
 ---> Running in a7940c60c8da
bin/dep: line 45: ./.dep: cannot execute binary file: Exec format error
The command '/bin/sh -c bin/dep ensure' returned a non-zero code: 126
```

Reply:
Thanks! I only tested on Windows and Linux and so I didn't notice that. I updated .dockerignore as you suggested.